install_config/uninstall-olm-using-ansible.adoc

```diff
@@ -0,0 +1,15 @@
+// This assembly is included in the following assemblies:
+//
+// * n/a
+
+:_content-type: ASSEMBLY
+[id="uninstall-olm-using-ansible"]
+= Uninstalling Operator Lifecycle Manager using Ansible
+include::_attributes/common-attributes.adoc[]
```

Comment:
As you asked in slack, this isn't correct and should be removed. It doesn't apply for 3.11, only 4.x versions.

Reply:
Addressed